docs/components/modeler/forms/configuration/forms-config-data-binding.md

```diff
@@ -0,0 +1,11 @@
+---
+id: forms-config-data-binding
+title: Data binding
+description: How data from the process interacts with your form
+---
+
+## Binding form fields to process data
+
+Each form element which allows data manipulation has a **Key** attribute, they are known as **Form Fields**. This attribute is used as an identifier to map data of the respective field (1) during initial loading of the form, and (2) during submission of the form.
```

Comment:
I don't think that we should capitalize everything that is in bold, only when it is also title case in the UI (and this is also discouraged). Same for many other words in bold throughout this doc. @christinaausley am I am right with this?